### DISCUSSION TITLE
Feature suggestion: add support for typing Japanese names in Chinese pinyin / wubi

### DISCUSSION THREAD

**Discussion by shiyuwang-jamk** [snomiao/rime-snomiao#4] [Ideas]
Like Japanese polititians and celebrities.

**Comment by snomiao** [comment]
need more examples

**Comment by shiyuwang-jamk** [comment]
How about this: https://github.com/wainshine/Chinese-Names-Corpus/blob/master/Japanese_Names_Corpus/Japanese_Names_Corpus%EF%BC%8818W%EF%BC%89.txt

**Reply by shiyuwang-jamk** [reply]
or a dictionary: https://mdict.org/post/riben-renming-diming/

**Comment by shiyuwang-jamk** [comment]
Some scenarios:
- Type Japanese gana -> gets name in kanji
- Type Chinese pinyin / wubi -> gets name in 
    - kanji or 
    - Japanese hiragana, or romaji
    - their English transliteration, for searching in English literatures.

**Reply by shiyuwang-jamk** [reply]
- Types romaji or English trasliteration in given name / surname order:
-> gets the kanji in surname / given name order
